src/style.css

```diff
@@ -0,0 +1,72 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+body {
+  font-family: Avenir, Arial, sans-serif;
+  font-weight: 300;
+}
+
+.wrapper {
+  position: relative;
+  margin: 20px 0;
+}
+
+.find {
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  height: 70px;
+  padding-left: 62px;
+  outline: none;
+  transition: box-shadow 0.2s;
+  font-family: Avenir, Arial, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+  font-size: 16px;
+}
+
+.find:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.find:focus {
+  font-weight: 900;
+  box-shadow: none;
+  border-radius: 3px;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.find-small {
+  font-size: 14px;
+  height: 42px;
+  padding-left: 33px;
+  font-weight: 400;
+}
+
+.loupe {
+  position: absolute;
+}
+
+.loupe-big {
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+  bottom: 26px;
+}
+
+.loupe-small {
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+  bottom: 16px;
+}
```

Comment:
you should have default loupe and big/small. Not big and small at the same time. One of them is just loupe without big/small word. Just like with `find` class in this file